### PR TITLE
use proper singularized version of species in the sample

### LINKED ISSUE #77: String Pluralization library should be case insensitive
For example, it's incorrectly singularizing `species` to `specy`

### PR DESCRIPTION
**Pull request by marwanad**
Should be 🍏  once it picks up the new plugin snapshot.

Closes #77

### CODE REVIEW

**Comment by marwanad** [comment]
Oh, sample now uses pojo now so no need for that.

I'll just verify that the generated `DroidDetails` has Species instead and update the PR with the README changes.

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ ext {
 
   dep = [
       androidPlugin: 'com.android.tools.build:gradle:2.3.0-beta3',
-      apolloPlugin: "com.apollographql.android:gradle-plugin:0.1.0",
+      apolloPlugin: "com.apollographql.android:gradle-plugin:0.2.0-SNAPSHOT",
```
Comment (marwanad) [inline build.gradle]: Is it worth having some kind of a `dev` branch that we build snapshots off instead and make that our target branch? and keep master for the releases?

File: README.md [hunk 2]
```diff
@@ -134,16 +157,37 @@ public final class DroidDetails implements Query<Operation.Variables> {
...
       private @Nullable String name;
 
       private @Nullable String classification;
 
-      public Specy(ResponseReader reader) throws IOException {
+      public Species(ResponseReader reader) throws IOException {
         MAPPER.map(reader, this);
       }
 
+      public Species(@Nonnull String id, @Nullable String name, @Nullable String classification) {
```
Comment (marwanad) [inline README.md]: will get rid of this. I just copied off the generated class.

File: README.md [hunk 3]
```diff
@@ -134,16 +157,37 @@ public final class DroidDetails implements Query<Operation.Variables> {
         }
       };
 
+      public static final Creator CREATOR = new Creator() {
```
Comment (marwanad) [inline README.md]: will get rid of this too.

File: build.gradle (hunk 1 above)
Comment (felipecsl) [inline build.gradle]: I'm not a fan of having development outside of master, we can keep master always pointing to the SNAPSHOTS I think. This shouldn't be pointing to 0.1.0 in the first place

Comment (marwanad) [inline build.gradle]: Ahh, fair enough. That solves it.

**Comment by sav007** [comment]
@marwanad but does compiler correctly generate interface names for Species ?
If not then example will mislead user as the outcome of generation is different

**Comment by marwanad** [comment]
@sav007 It does for 0.2-SNAPSHOT, not for 0.1.0.

Shall we remove that/update the README to use the snapshot version?

**Pull request merged**